tests/Doctrine/Tests/Models/VersionedManyToOne/Article.php

```diff
@@ -14,7 +14,7 @@
 
 /**
  * @Entity
- * @Table(name="versioned_many_to_one_article")
+ * @Table(name="ver_m2o_article") // fix Max identifier length for OracleDB
```

Comment:
This comment isn't necessary here. It could be applied to any other object name as well.